packages/core/contracts/cross-chain-oracle/chain-adapters/ParentMessengerBase.sol

```diff
@@ -0,0 +1,80 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+contract ParentMessengerBase is Ownable {
+    uint256 public chainId;
+
+    address public childMessenger;
+
+    address public oracleHub;
+    address public governorHub;
+
+    address public oracleSpoke;
+    address public governorSpoke;
+
+    uint32 public defaultGasLimit = 5_000_000;
+
+    modifier onlyPrivilegedCaller() {
```

Comment:
Should we rename this modifier to indicate that its only a Hub contract that can call?

Reply:
sure. I changed it to `onlyHubContract`